react/features/app/components/AbstractApp.js

```diff
@@ -134,11 +137,13 @@ export class AbstractApp extends Component {
 
         if (route) {
             return (
-                <Provider store = { this._getStore() }>
-                    {
-                        this._createElement(route.component)
-                    }
-                </Provider>
+                <I18nextProvider i18n = { i18n }>
```

Comment:
Well, ok, but this is how react-i18next should be implemented following their docs, what if in a newer version that provider changes. Yes we can add it then, but I'm just in a doubt about this, have no problem changing it.